### PR TITLE
android serializer

### PR DESCRIPTION
**Pull request by bruno-garcia**

### CODE REVIEW

File: sentry-android/src/main/java/io/sentry/internal/AndroidSerializer.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package io.sentry.internal;
+
+import io.sentry.ISerializer;
+import io.sentry.SentryEvent;
+
+public class AndroidSerializer implements ISerializer {
+  @Override
+  public SentryEnvelope DeserializeEnvelope(String envelope) {
```
Comment (marandaneto) [inline sentry-android/src/main/java/io/sentry/internal/AndroidSerializer.java]: name it as camelCase

File: sentry-android/src/main/java/io/sentry/internal/AndroidSerializer.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    return null;
+  }
+
+  @Override
+  public String Serialize(SentryEvent event) {
```
Comment (marandaneto) [inline sentry-android/src/main/java/io/sentry/internal/AndroidSerializer.java]: name it as camelCase

File: sentry-core/src/main/java/io/sentry/ISerializer.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package io.sentry;
+
+import io.sentry.internal.SentryEnvelope;
+
+public interface ISerializer {
+  SentryEnvelope DeserializeEnvelope(String envelope);
```
Comment (marandaneto) [inline sentry-core/src/main/java/io/sentry/ISerializer.java]: name it as camelCase

File: sentry-android/src/main/java/io/sentry/internal/AndroidSerializer.java (hunk 1 above)
Comment (bruno-garcia) [reply sentry-android/src/main/java/io/sentry/internal/AndroidSerializer.java]: I'll never learn

**Pull request merged**